src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/ValidateCredentials.java

```diff
@@ -46,6 +46,6 @@ public boolean validate(String provider, String value) {
   }
 
   void onStop(@Observes ShutdownEvent ev) {
-    channel.shutdown();
+    if (channel != null) channel.shutdown();
```

Comment:
The channel is never initialized if no createCollection request hit the VM. Added a null check for proper shutdown of server.